docs/knowledge-base/release-management/release-management.mdx

```diff
@@ -0,0 +1,208 @@
+    - Pack the helm chart and push to registry (if a `helm` chart exists)
+
+**End State:**
+    - New release version based on SemVer
+    - master points to the new release
+    - NPM, Docker and Helm registries updated with the new release
+
+:::info
+The deployment part is described in the next section
+:::
+
+## Helming Strategy
+
+:::warning
+This strategy does not include the checks during development. 
+It assumes that the new released service is properly checked and its updated chart is good.
+All the development tests should happen before the service was released so in this flow we assume 
+that the chart is good and does not require deployment in order to know if it is a good chart
+:::
+
+![Helming Strategy](./images/helm-strategy.png)
+
+The project includes two different repositories that represent two different flows:
+
+1. **Chart Flow using "Helm-Charts" repo** - Handling the new charts and versions from the services and creation of new "Libot Version"
+
+2. **Values Flow using "Helm-Values" repo** - Handling the deployment in openshift based on the new "Libot Version" that was created in `helm-charts`
+
+### Explanation
+
+With this strategy, we separate the charts from the values.  
+
+"helm-charts" repo will contain only the chart. The structure of the helm.  
+"helm-values" repo will contain **only the values based on the relevant chart** 
+and the "Libot Version" it based on (a file named "libot-version.json" in the repo).
+The values files that represent namespaces of openshift should not be part of the structure 
+(as they only config files). They should be in a different repo.  
+That's way we earn:
+
+1. **Avoiding "trash" commits**  
+Imagine if we keep the values with the chart, Now if we want to change the log level from info to debug, 
+we will have to commit the change in the repo itself and this shouldn't be. 
+That way we will "trash" the repo with a lot with unnecessary commits that don't related to chart itself.
+
+2. **Allowing informative versions of helm-charts**  
+If we keep the chart and the values together, We will have a hard time to handle properly the release of helm-chart 
+duo to the many changes on the values that does not require a new release.
+From the other hand, there are changes in the values file that we want to change. Imagine that you want to change the default port to be 80. 
+This is a "structure" change because you want to change a default value so it does require a new release.
+So you won't know what requires a new release and what is not.
+With the separated values and charts, you will have just default values in the chart so every change in that values file will require new release.
+
+3. **One and only structure**
+Currently, the charts in the network can be different from the charts in Azure because the charts can be modified. 
+Using the new strategy, we make sure that there will be one source of truth and it sits in helm-charts outside the internal network and only there! 
```

Comment:
```suggestion
Currently, the charts in the network are different from the charts in Azure because they can be modified. Using the new strategy, we make sure that there will be one source of truth which is `helm-charts`.
```